[2022-08-25 20:48:41,517] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data.format_xlsx_to_csv_task scheduled__2021-01-10T06:00:00+00:00 [queued]>
[2022-08-25 20:48:41,557] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data.format_xlsx_to_csv_task scheduled__2021-01-10T06:00:00+00:00 [queued]>
[2022-08-25 20:48:41,562] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-25 20:48:41,564] {taskinstance.py:1239} INFO - Starting attempt 20 of 20
[2022-08-25 20:48:41,565] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-25 20:48:41,582] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_xlsx_to_csv_task> on 2021-01-10 06:00:00+00:00
[2022-08-25 20:48:41,590] {standard_task_runner.py:52} INFO - Started process 1317 to run task
[2022-08-25 20:48:41,600] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'h1b_lca_data', 'format_xlsx_to_csv_task', 'scheduled__2021-01-10T06:00:00+00:00', '--job-id', '323', '--raw', '--subdir', 'DAGS_FOLDER/upload_h1b_data_to_gcp.py', '--cfg-path', '/tmp/tmpttj_t9ep', '--error-file', '/tmp/tmpwx3kfn01']
[2022-08-25 20:48:41,601] {standard_task_runner.py:77} INFO - Job 323: Subtask format_xlsx_to_csv_task
[2022-08-25 20:48:41,707] {logging_mixin.py:109} INFO - Running <TaskInstance: h1b_lca_data.format_xlsx_to_csv_task scheduled__2021-01-10T06:00:00+00:00 [running]> on host c8cac1ea6d9c
[2022-08-25 20:48:41,765] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-08-25 20:48:41,790] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=h1b_lca_data
AIRFLOW_CTX_TASK_ID=format_xlsx_to_csv_task
AIRFLOW_CTX_EXECUTION_DATE=2021-01-10T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-10T06:00:00+00:00
[2022-08-25 20:48:43,066] {logging_mixin.py:109} WARNING - WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
[2022-08-25 20:48:57,214] {local_task_job.py:154} INFO - Task exited with return code Negsignal.SIGKILL
[2022-08-25 20:48:57,305] {taskinstance.py:1277} INFO - Marking task as FAILED. dag_id=h1b_lca_data, task_id=format_xlsx_to_csv_task, execution_date=20210110T060000, start_date=20220825T204841, end_date=20220825T204857
[2022-08-25 20:48:57,458] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
